movies/src/components/common/button/Button.js

```diff
@@ -2,15 +2,24 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 export default function Button(props) {
-  const { className, title } = props;
+  const {
+    className, title, handleClick, type,
+  } = props;
+
   return (
-    <button className={className} type="button">
+    <button className={className} type={type} onClick={handleClick}>
```

Comment:
I think you can disable rule `button-has-type` in global eslint config

Reply:
Thanks, Fixed.